### PR TITLE
Disable pds appview indexing

### PR DESCRIPTION
**Pull request by dholms**
Follow up to https://github.com/bluesky-social/atproto/pull/1644 & precursor to merging https://github.com/bluesky-social/atproto/pull/1198 

This PR removes all appview indexing logic from the PDS.

### CODE REVIEW

File: packages/dev-env/src/mock/index.ts [hunk 1]
```diff
@@ -186,30 +186,30 @@ export async function generateMockSetup(env: TestNetworkNoAppView) {
...
-          cts: new Date().toISOString(),
-        },
-      ])
-      .execute()
-  }
+  // const ctx = env.pds.ctx
+  // if (ctx) {
+  //   await ctx.db.db
+  //     .insertInto('label')
+  //     .values([
```
Comment (devinivy) [inline packages/dev-env/src/mock/index.ts:193]: Can we nix this code?

File: packages/pds/src/app-view/api/app/bsky/unspecced.ts [hunk 2]
```diff
@@ -43,18 +43,6 @@ export default function (server: Server, ctx: AppContext) {
       }
     },
   })
-
-  server.app.bsky.unspecced.applyLabels({
```
Comment (devinivy) [inline packages/pds/src/app-view/api/app/bsky/unspecced.ts:47]: I expect we'll want to remove the use of this endpoint in `packages/bsky/src/auto-moderator/index.ts`.

File: packages/pds/src/db/migrations/20230922T033938477Z-remove-appview.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  await db.schema.dropTable('post').execute()
+  await db.schema.dropTable('profile_agg').execute()
+  await db.schema.dropTable('profile').execute()
+  await db.schema.dropTable('repost').execute()
+  await db.schema.dropTable('subscription').execute()
+  await db.schema.dropTable('suggested_follow').execute()
+  await db.schema.dropTable('view_param').execute()
+}
+
+export async function down(db: Kysely<unknown>): Promise<void> {
```
Comment (devinivy) [inline packages/pds/src/db/migrations/20230922T033938477Z-remove-appview.ts:27]: I think this check warning may continue to bug us if we don't handle it here 🙃 
```suggestion
export async function down(_db: Kysely<unknown>): Promise<void> {
```

**Review by devinivy — APPROVED** [review]
Looks solid! 👋 goodbye in-process appview, it's been real.

File: packages/dev-env/src/mock/index.ts (hunk 1 above)
Comment (dholms) [reply packages/dev-env/src/mock/index.ts]: nixed the pds-only dev-env & switched this code to insert the labels into the appview 👌

File: packages/pds/src/app-view/api/app/bsky/unspecced.ts (hunk 2 above)
Comment (dholms) [reply packages/pds/src/app-view/api/app/bsky/unspecced.ts:47]: yup - wiped out the whole schema 👌

**Pull request merged**